src/task.c

```diff
@@ -561,7 +561,10 @@ static void JL_NORETURN throw_internal(jl_task_t *ct, jl_value_t *exception JL_M
     ptls->io_wait = 0;
     // @time needs its compile timer disabled on error,
     // and cannot use a try-finally as it would break scope for assignments
-    jl_measure_compile_time[ptls->tid] = 0;
+    // We blindly disable compilation time tracking here, for all running Tasks, even though
+    // it may cause some incorrect measurements. This is a known bug, and is being tracked
+    // here: https://github.com/JuliaLang/julia/pull/39138
+    jl_atomic_store_release(&jl_measure_compile_time_enabled, 0);
```

Comment:
@vtjnash can you help me with this? What's the difference between `jl_atomic_store` and `jl_atomic_store_release`? The rest of the code here seems to use `_release`, but i'm not sure what i should do here.

Since I'm using `jl_atomic_load()` for this variable everywhere else, should i just use `jl_atomic_store`?

Reply:
It specifies restrictions on what must happen to other memory:

relaxed -> no other constraints
release (store) -> all stores before this must occur before this is visible to any processor
acquire (load) -> all loads after this must occur after this is visible to this processor
seq_cst -> (store) all stores and loads must happen in program order before/after this OR (load) all loads must happen in program order before/after this

We are probably okay with `relaxed` on this everywhere, since I don't think we are attempting to synchronize any other operations

Reply:
okay thanks, i'll change everything to relaxed everywhere. 🏖️